jenkins-pipelines/longevity-mv-si-24h-gce.jenkinsfile

```diff
@@ -1,12 +0,0 @@
-#!groovy
```

Comment:
Is it intentional file deletion? It doesn't look related to the goal of the PR.

Reply:
Squashed all commits to a single commit with clear message description.